### ISSUE TITLE
Latest release on NPM does not include #8

### ISSUE THREAD

**Issue by forivall** [drgx/http-to-curl#9]
It appears that the build was not run the last time this package was published, and so the bundle.js includes the "console.log" in the `console.log(`${chalk.black.bgYellow.bold(" http-to-curl ")}\n  ${e}\n  `)` in `curlGenerator()` - #8 should have corrected this.

**Comment by forivall** [comment]
Also, FYI, my usage was with `got`, which works perfectly with this package:
```
      got.get(`apis/${api.id}/implementations`, {
        ...baseGotOptsJson,
        hooks: {
          beforeRequest: [(options) => {
            console.log(httpToCurl.curlGenerator(options))
            return options
          }]
        }
      })
```

**Comment by drgx** [comment]
hi @forivall , thanks for reporting and sorry for slow response

I just published the new version `v 1.4.3` Please try again.
